Mathlib/CategoryTheory/ObjectProperty/FunctorCategory/PreservesLimits.lean

```diff
@@ -24,10 +24,29 @@ namespace CategoryTheory
 
 open Limits
 
-variable {J C : Type*} (K K' : Type*) [Category K] [Category K'] [Category J] [Category C]
+variable {J J' C D : Type*} (K K' : Type*)
+  [Category K] [Category K'] [Category J] [Category J'] [Category C] [Category D]
 
 namespace ObjectProperty
 
+variable {K} in
+/-- The property of objects in the functor category `J ⥤ C`
+which preserves the limit of a functor `F : K ⥤ J`. -/
+abbrev preservesLimit (F : K ⥤ J) : ObjectProperty (J ⥤ C) := PreservesLimit F
+
+@[simp]
+lemma preservesLimit_iff (F : K ⥤ J) (G : J ⥤ C) :
+    preservesLimit F G ↔ PreservesLimit F G := Iff.rfl
+
```

Comment:
Since you added `congr_preservesLimitsOfShape`, might as well add the similar lemma for individual diagrams:

```suggestion

lemma congr_preservesLimit {F F' : K ⥤ J} (e : F ⟶ F') [IsIso e] :
    preservesLimit (C := C) F = preservesLimit (C := C) F' := by
  ext G
  simp_rw [preservesLimit_iff]
  exact ⟨fun h => preservesLimit_of_iso_diagram _ (asIso e),
    fun h => preservesLimit_of_iso_diagram _ (asIso e).symm⟩
   
```
or maybe the `Iso` variant:

```suggestion

lemma congr_preservesLimit {F F' : K ⥤ J} (e : F ≅ F'):
    preservesLimit (C := C) F = preservesLimit (C := C) F' := by
  ext G
  simp_rw [preservesLimit_iff]
  exact ⟨fun h => preservesLimit_of_iso_diagram _ e,
    fun h => preservesLimit_of_iso_diagram _ e.symm⟩

```

and similarly for colimits.